<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultTab</class>
 <widget class="QWidget" name="ResultTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBoxRowSelectionBehavior">
     <property name="title">
      <string>选中某一行时，将画布移至</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonFeature">
        <property name="text">
         <string>要素</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonError">
        <property name="text">
         <string>错误</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButtonDontMove">
        <property name="text">
         <string>不移动</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="checkBoxHighlight">
        <property name="text">
         <string>高亮选中要素</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="openWidge" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnOpenAttributeTable">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/themes/default/mActionFromSelectedFeature.svg</normaloff>:/images/themes/default/mActionFromSelectedFeature.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOpenAttributeTable">
        <property name="text">
         <string>在属性表中打开所选要素</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidgetErrors">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>图层</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>错误原因</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>坐标</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>对象信息</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>修复</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="text">
         <string>导出</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>280</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelErrorCount">
        <property name="text">
         <string>错误数目： 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="fixWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="2">
       <widget class="QLabel" name="labelErrorResolutionSettings">
        <property name="text">
         <string>设置修复方法</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>修复方法</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnErrorResolutionSettings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>设置修复方法</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/themes/default/mActionOptions.svg</normaloff>:/images/themes/default/mActionOptions.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnFixWithDefault">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>修复</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/images/themes/default/mIconSuccess.svg</normaloff>:/images/themes/default/mIconSuccess.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="btnFix">
        <property name="text">
         <string>修复</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="3">
       <widget class="QProgressBar" name="progressBarFixErrors">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="4">
       <widget class="QComboBox" name="mComboBox">
        <property name="whatsThis">
         <string>select fix method</string>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
